Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/User.java

```diff
@@ -26,19 +26,23 @@
 import javax.persistence.OneToMany;
 import javax.persistence.Table;
 
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.GenericField;
 import org.kitodo.data.database.persistence.UserDAO;
 
 @Entity
 @Table(name = "user")
```

Comment:
Is an `@Indexed(index = "kitodo-user")` annotation not missing like in the other bean files?

Reply:
Users are not indexed because otherwise you cannot log in before the index is created. Then you cannot create the index because you cannot log in. This was not planned at the very beginning but is now the case.

Reply:
I see. One of the chicken and egg problems at least and even an issue of privacy restrictions.